integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -262,7 +252,7 @@ class CarbonFileMetastore extends CarbonMetaStore {
                       .inferSchema(identifier, false),
             dbName, tableName)
       }
-
+      schemaRefreshTime = System.currentTimeMillis()
```

Comment:
you can initialize at line 233, no need of this line here

Reply:
changed